gc/base/standard/ObjectScanner.hpp

```diff
@@ -190,6 +191,10 @@ class GC_ObjectScanner : public MM_BaseVirtual
 	MMINLINE static bool isIndexableObject(uintptr_t flags) { return (0 != (indexableObject & flags)); }
 
 	MMINLINE bool isIndexableObject() { return (0 != (indexableObject & _flags)); }
+
+	MMINLINE static bool isIndexableObjectNoSplit(uintptr_t flags) { return (0 != (indexableObjectNoSplit & flags)); }
```

Comment:
I do not see any callers of these new APIs (reviewing on my phone so the screen is tiny).  Am I missing something?